panel/panel.vala

```diff
@@ -952,22 +952,22 @@ public class Panel : Budgie.Toplevel
         }
         this.expanded = expanded;
         if (!expanded) {
-            if (wm_proxy != null) {
+            /*if (wm_proxy != null) {
```

Comment:
Actually, one thing that I just thought. What about modifying the function *show_popover* of the PopoverManager, to allows us to specify if we want the previous active window to be made active again? We could add a  boolean argument to select the desired behavior. If we were to call:
```vala
popover_manager.show_popover(btn, false);
```
Once the popover associated with *btn* was closed, nothing would happen. 
To avoid breaking compatibility in the vala interface, we could set this boolean value to default to true (to make the previous active window active again). For the C interface, this would require exposing a function with a different name.

Would such a solution be fine with you?